src/panels/lovelace/cards/hui-glance-card.ts

```diff
@@ -59,7 +59,7 @@ export class HuiGlanceCard extends LitElement implements LovelaceCard {
       includeDomains
     );
 
-    return { entities: foundEntities };
+    return { type: "glance", entities: foundEntities };
```

Comment:
Why do we need type here?

Reply:
Because we should return a valid stub config. Adding type to return type also helps enforce ethat. The fact that the card picker doesn't need it is an implementation detail that we should not leak into our stub configs.

Reply:
Then we should add it to the rest.